src/database/entities/millions-campaign-member.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { Column, CreateDateColumn, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn, Relation, UpdateDateColumn } from 'typeorm';
+
+import { MillionsCampaignEntity } from '@app/database';
```

Comment:
This is invalid. When cross-referencing files between the same virtual alias, you must use absolute path and not the magic one. This would likely fail to compile on production environments.